content/docs/guide/map_submission/map_zoning.md

```diff
@@ -9,60 +9,413 @@ weight: 3
 ---
 
 # Introduction
-
-Zoning is the process of defining the start, end, checkpoint, and stage areas of tracks on a map. This is required for all maps that are submitted to Momentum Mod. Zoning can be done in-game or in Hammer, both methods are described in this guide.
+Zones allow players to measure and submit times to the leaderboards.  
+This guide focuses on zoning using Momentum Mod's **in-game tool**.   
+Zoning using **Hammer** is briefly covered in [this section](#hammer-zoning)
 
 # General Guidelines
+- Zones should **always** use teleport destinations **defined by the mapper** when possible.
```

Comment:
Zones with teleport destinations should **always** use entities **defined by the mapper** when possible.